apps/web/app/(navigation)/CreateNewBudget.styled.tsx

```diff
@@ -150,6 +150,17 @@ export const TabsStyled = styled(Tabs.Root)`
   }
 `;
 
+export const ErrorMessageWrapper = styled.div`
+  position: absolute;
+  z-index: 100;
+  margin-top: 55px;
+  margin-left: -6px;
+  width: 315px;
+  ${device.tablet} {
+    width: 460px;
+  }
+`;
```

Comment:
I don't like the idea of setting it to fixed width. What if design changes and/or new break points are added? I think it is better to use `width = 100%` for such nested elements, so it always works well. You can do few changes in styles and it will work fine, and there will be also fewer lines of code

```suggestion
export const ErrorMessageWrapper = styled.div`
  width: 100%;
  margin-bottom: -45px;
  position: relative;
  margin-top: -22px;
`;
```

output:

https://github.com/intive/patronage2023-js/assets/80450315/1fe794d5-0754-4681-b498-a83249ae9a7e